doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+This assumption can be justified physically by considering the structure of a galaxy.
+In general the populations of stars in the bulge and disk are different, as well as perhaps additional populations of star forming regions, each with its own distinct SED.
+However, the actual colors in each population are not that different, which is why parametric bulge-disk models (Sersic models, Spergel models, Gaussian mixture models, etc.) have been successfully used in deblending and photometric measurement.
+
+The other assumption that is currently made in the science pipelines models is that each component is monotonically decreasing from a peak.
+Again, thinking of galactic structure this is usually a reasonable assumption as both the bulge and disk are brightest in the center, and even spiral arms generally emanate from a peak location.
+There are exceptions, however, like nebula and irregular galaxies that are much less organized and not well constrained by a component-like model with a center.
+
+The other main astrophysical effect that is overlooked by these assumptions is extinction by gas and dust, which can cause reddening over a single component.
+While it is outside of the discussion in this tutorial, we have observed reddening residuals in edge-on galaxies and it is currently an active area of research to better model galactic (and intergalactic) dust using *scarlet*.
+
+Because the *scarlet* model is multi-band, and the observations are not required (or recommended) to be matched to a common PSF, the model itself must exist in a space that can be convolved into the observed seeing in each band, which we will refer to as *model space*.
+In order to be sufficiently well sampled we currently use a PSF for the model space that is a circular Gaussian with ``σ=0.8`` pixels, ensuring that our model image is well-sampled but smaller than the observed PSF in each observed band.
+
+.. note::
+    It is possible that a single visit may have an under-sampled PSF, in which case one can expect artifacts and erratic behavior from the *scarlet* models.
+    This problem is not isolated to the deblender, as under-sampled PSFs cause problems in many other places further upstream and will cause much larger issues than the deblender results.
+    However, there are currently no plans to use *scarlet* on single visits and it is expected that co-adds will always be well-sampled.
+
+.. _model_init:
+
+Initialization
+^^^^^^^^^^^^^^
+
+One way to think of the initial *scarlet* models is that they are basically the same as the templates generated by the single band deblender (see `single-band_deblending`_) with one key difference: because the *scarlet* models are obtained using an iterative optimization procedure, using the assumption of monotonicity from the peak location is a well-defined and useful (perhaps necessary) constraint.
+
+For *scarlet* we have chosen the *𝛘²* (inverse variance-weighted or Kaiser) co-add to initialize our morphologies.
```

Comment:
Briefly explain this, and elaborate on whether this a dataset in the butler, or something you generate on-the-fly?